testing/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/ivy/IvyFileModule.groovy

```diff
@@ -149,6 +149,16 @@ class IvyFileModule extends AbstractModule implements IvyModule {
         return this
     }
 
+    @Override
+    Module eachVariant(@DelegatesTo(value = VariantMetadataSpec.class, strategy = Closure.DELEGATE_FIRST) Closure<?> action) {
```

Comment:
These test fixture methods are no longer used by this PR, but will be useful later. Better to leave them